src/app/[locale]/page.tsx

```diff
@@ -1,6 +1,11 @@
 import { useTranslations } from 'next-intl';
-
+import Button from '../../Components/Buttons/Button';
 export default function Home() {
 	const t = useTranslations();
-	return <div>{t('loginPage.title')}</div>;
+	return <div>{t('loginPage.title')}
+	<Button text="Login" variant='primary' />
+	<Button text="Sign up" variant='secondary' />
```

Comment:
Creating a component doesn't mean that you should integrate them to pages. Actually quite the opposite, you never integrate a component into another component/page in the same PR as you created them. Also, that's why we have the Storybook.

If you want to see how your component looks, you should create the story for it (stories.ts file) and run `npm run sb`.

Please remove these components from the page and create the story accordingly.